arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/ar/ArcGISArView.kt

```diff
@@ -87,7 +87,7 @@ final class ArcGISArView : FrameLayout, DefaultLifecycleObserver, Scene.OnUpdate
      *
      * @since 100.6.0
      */
-    private val initialTransformationMatrix = TransformationMatrix.createIdentityMatrix()
+    var initialTransformationMatrix: TransformationMatrix = TransformationMatrix.createIdentityMatrix()
```

Comment:
Just to double check - this is a non-nullable property, correct?

Reply:
Correct. It's either the identity matrix or one the user has set.